src/utils/http-common.js

```diff
@@ -2,10 +2,12 @@ import axios from "axios";
 
 const { NODE_ENV } = process.env;
 
-const baseURL =
-  NODE_ENV === "development"
-    ? "http://localhost:3004/api"
-    : "https://amexcsrapi.azurewebsites.net/api";
+// const baseURL =
```

Comment:
@ivwu Make sure this is re-enabled before merging

Reply:
```
const baseURL =  NODE_ENV === "development"
    ? "http://localhost:3004/api"
    : "https://amexcsrapi.azurewebsites.net/api";
```

Reply:
should i re-enable it right now or when the backend is done completing the model for customers

Reply:
You can re-enable it now. We do want to still keep the production site connected to the production backend at all times, even if some backend features are incomplete.